src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -644,6 +650,7 @@ typedef struct {
     // If unset GENERATED_CERTIFICATE_BITS will be used
     INT32 generatedCertificateBits;
 
+    IceSetInterfaceFilterFunc setFilterFn;
```

Comment:
I would call this `iceSetInterfaceFilterFunc`

Reply:
Typo at both locations. Changing it now.